requirements.txt

```diff
@@ -14,3 +14,8 @@ netifaces==0.11.0; sys.platform == "win32"
 WMI==1.5.1; sys.platform == "win32"
 ./granulate-utils/
 humanfriendly==10.0
+wheel
```

Comment:
Why do we need `wheel` in the agent? (not only in dev or in one of the Dockerfile steps?)